### PR TITLE
Add indicator endpoints

### PR DESCRIPTION
**Pull request by aitzkovitz**
- add models, consts, methods for indicators endpoints, also replace funky single quotes
- add tests, some model fixes

### CODE REVIEW

**Comment by aitzkovitz** [comment]
Won't be merging this until technical indicators out of course

**Review by morningvera — COMMENTED** [review]
This looks great, thanks for contributing it! I didn't see any glaring issues but I'll wait to approve until we've released the indicators API

File: rest/models/indicators.go [hunk 1]
```diff
@@ -0,0 +1,419 @@
+package models
+
+// GetSMAParams is the set of parameters for the GetSMA method.
+type GetSMAParams struct {
+	// The ticker symbol of the stock/equity.
+	Ticker string `validate:"required" path:"ticker"`
+
+	// The size of the timespan multiplier for the underlying aggregates.
+	Multiplier *int `query:"multiplier"`
```
Comment (morningvera) [inline rest/models/indicators.go:9]: did y'all remove this param from the API? not seeing it in the docs anymore

Comment (aitzkovitz) [reply rest/models/indicators.go:9]: ah nice catch

**Review by morningvera — APPROVED** [review]
lgtm! looks like CodeQL is complaining about something unrelated to this PR so i'm going to disregard it for now

**Pull request merged**